README.md

```diff
@@ -51,15 +39,27 @@ BLT provides an automation layer for testing, building, and launching Drupal 8 a
     ```
     $ composer install
     ```
```

Comment:
Need to run `blt  blt:init:settings` to generate a local.setttings.php file for each site.